CMakeLists.txt

```diff
@@ -292,8 +295,23 @@ elseif (BUNDLE STREQUAL "miner")
 	set(ETHKEY OFF)
 	set(MINER ON)
 	set(ETHASHCL ON)
-elseif (BUNDLE STREQUAL "release")
-	set(SERPENT OFF)
+elseif (BUNDLE STREQUAL "default")	# development builds
+	set(SERPENT ${DECENT_PLATFORM})
+	set(SOLIDITY ON)
+	set(USENPM OFF)
+	set(GUI ON)
+	set(TOOLS ON)
+	set(TESTS ON)
+	set(FATDB ON)
+	set(ETHASHCL ON)
+	set(EVMJIT ON)
```

Comment:
runtime check? in any case this won't be active until the buildbots are switch to the bundle called default.